### ISSUE TITLE
Check version of compiler before starting to compile code

### ISSUE THREAD

**Issue by mfrey** [des-testbed/libara#90]
The build system really drives me crazy. If you compile the on knecht and don't replace the value of the CXX variable with gcc/g++ over clang. You get odd issues that src/libara.so can't be build because of a missing target of Exception.o - if we check the required compiler (clang >= 3.2 and gcc >= 4.8) this issue might not occur anymore (I'm going to replace the CXX before I get into this deadlock). A ``make clean`` does not help  and removing the ``out`` does also not help at all. This is so frustrating!

**Comment by mfrey** [comment]
This does not seem to be an issue anymore. I will close this and mark it as an wontfix.
